### ISSUE TITLE
Support for Parties 3

### ISSUE THREAD

**Issue by ChrissW-R1** [PikaMug/Unite#2]
Hi with Parties in version 3 (especially 3.2.7), I get the following issue on server startup:
> IllegalStateException: PartiesAPI has not been initialized

```[21:59:54] [Server thread/INFO]: [Unite] Enabling Unite v1.1.3
[21:59:54] [Server thread/ERROR]: [Unite] [Party] There was an error hooking Parties - check to make sure you're using a compatible version!
[21:59:54] [Server thread/WARN]: java.lang.reflect.InvocationTargetException
[21:59:54] [Server thread/WARN]: 	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:79)
[21:59:54] [Server thread/WARN]: 	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[21:59:54] [Server thread/WARN]: 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:483)
[21:59:54] [Server thread/WARN]: 	at Unite-1.1.3.jar//me.pikamug.unite.Unite.hookProvider(Unite.java:66)
[21:59:54] [Server thread/WARN]: 	at Unite-1.1.3.jar//me.pikamug.unite.Unite.hookProviders(Unite.java:57)
[21:59:54] [Server thread/WARN]: 	at Unite-1.1.3.jar//me.pikamug.unite.Unite.onEnable(Unite.java:30)
[21:59:54] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
[21:59:54] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[21:59:54] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:542)
[21:59:54] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:565)
[21:59:54] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:479)
[21:59:54] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:279)
[21:59:54] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1100)
[21:59:54] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305)
[21:59:54] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[21:59:54] [Server thread/WARN]: Caused by: java.lang.IllegalStateException: PartiesAPI has not been initialized
[21:59:54] [Server thread/WARN]: 	at Parties-3.2.7.jar//com.alessiodp.parties.api.Parties.getApi(Parties.java:22)
[21:59:54] [Server thread/WARN]: 	at Unite-1.1.3.jar//me.pikamug.unite.api.objects.plugins.PartyProvider_Parties.<init>(PartyProvider_Parties.java:43)
[21:59:54] [Server thread/WARN]: 	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
[21:59:54] [Server thread/WARN]: 	... 14 more
[21:59:54] [Server thread/INFO]: [Unite] [Party] mcMMO found: Waiting
```

Environment:
- PaperMC (1.19.2-307)
- Unite (1.1.3)
- [Parties](https://www.spigotmc.org/resources/parties-an-advanced-parties-manager.3709/) (3.2.7)

**Comment by PikaMug** [comment]
Unite 1.1.3 is built against Parties 3.1.8, but it seems more likely you have Parties installed and it's failing to initialize/enable. Will take a closer look later.

**Comment by AlessioDP** [comment]
Can you post the entire server log?
`PartiesAPI has not been initialized` means that Parties has not been completely initialized (maybe a startup error?).

**Comment by PikaMug** [comment]
Think I found the issue on my end, but a dependency of a dependency prevents me from doing anything until https://maven.sk89q.com/repo/ is back up. 

EDIT: or until https://github.com/elBukkit/MagicPlugin/pull/1159 is merged.

**Comment by PikaMug** [comment]
Should be fixed in d7a127d2c7bcba53d17584579758279a9141e252
